istio/manifest.json

```diff
@@ -24,7 +24,9 @@
   "integration_id": "istio",
   "assets": {
     "monitors": {},
-    "dashboards": {},
+    "dashboards": {
+      "Istio base dashboard": "assets/dashboards/istio_dashboard.json"
```

Comment:
(nit) Should we name it similarly to other dashboards ? 

Most dashboard are named `XXX Overview` and the file is called `assets/dashboards/overview.json`.

![image](https://user-images.githubusercontent.com/49917914/65322194-65c56480-dba6-11e9-8c66-3a712208ec69.png)